bin/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+SSH_SECRETS_DIR=${SSH_SECRETS_DIR:-/etc/feedhenry/gitlab-shell}
+AUTHORIZED_KEYS_FILE=${AUTHORIZED_KEYS_FILE:-/home/git/.ssh/authorized_keys} 
+
+echo "Initializing authorized_keys file"
+cat $SSH_SECRETS_DIR/repoadmin-id-rsa-pub > $AUTHORIZED_KEYS_FILE
```

Comment:
Does this mean the file will be reset any time the container restarts or is rescheduled?